dangerzone/cli.py

```diff
@@ -104,11 +104,11 @@ def cli_main(
 
 
 def setup_logging() -> None:
-    if getattr(sys, "dangerzone_dev", True):
-        fmt = "%(message)s"
+    if getattr(sys, "dangerzone_dev", False):
+        fmt = "[%(levelname)-5s] %(message)s"
         logging.basicConfig(level=logging.DEBUG, format=fmt)
     else:
-        logging.basicConfig(level=logging.ERROR, format=fmt)
+        logging.basicConfig(level=logging.INFO, format="%(message)s")
```

Comment:
Looks good. One minor question though: why don't we print the error level in the basic configuration, and we print just the message?

Reply:
This is the "logging" that end-users are supposed to see when they run the CLI version of dangerzone. So I don't think we should show them 'INFO before all the messages.

But you're right. Probably for `WARN` and `ERROR`, we want to label them as such. I'm looking at way of doing that.

Reply:
I have fixed this now in https://github.com/freedomofpress/dangerzone/pull/301/commits/565ad300ceea509ee7f6a93099a3e9cdf5b5cc79. You can test it out by setting `logger.setLevel()` to `logging.DEBUG` and seeing that it prints a `DEBUG` line:

```
[...]
Converting document to safe PDF
DEBUG Marking doc 4LAzqc as 'converting'
> /usr/bin/podman run --network none -u dangerzone  [...]
```

Reply:
What do you think about this solution?

Reply:
Hm, I think the types of users that will see these logs will probably be the users who won't mind seeing the `INFO` log level. So, I wouldn't add this extra complexity. If it's an eyesore for you, sure, feel free to merge it.

Reply:
Okey dokey!

Reply:
Whoops. My mistake: I accidentally implemented this logic for a dev environment and not the final version (the one users run). Maybe this was what confused you.